src/olympia/editors/tests/test_views.py

```diff
@@ -1980,12 +1858,13 @@ def test_breadcrumbs(self):
         self._test_breadcrumbs(expected)
 
     def test_breadcrumbs_unlisted_addons(self):
-        self.addon.update(is_listed=False)
+        self.addon.update(is_listed=False, status=amo.STATUS_PUBLIC)
```

Comment:
Yeah, it's necessary to make it in the state we are now, it might change once we make some progress but I wanted to be in a stable state first.